tests/connect/standalone-x509-extract_username-002.phpt

```diff
@@ -11,7 +11,7 @@ parse_url() tests must be reimplemented (PHPC-1177)
 <?php
 require_once __DIR__ . "/../utils/basic.inc";
 
-$SSL_DIR = realpath(__DIR__ . '/../../scripts/ssl/');
+$SSL_DIR = realpath(getenv('SSL_DIR'));
```

Comment:
While you're modifying these lines, what do you think about defining an SSL_DIR constant in `basic.inc` and just getting rid of this? Something like:

```php
define('SSL_DIR', realpath(getenv('SSL_DIR') ?: ''));
```

`realpath()` is happy to take `false` so maybe the ternary is superfluous. It will all resolve to the CWD and ultimately lead to a bad path when we actually try to refer to certs in a test. For that reason, I think we're probably OK doing without the change to `skip_if_not_ssl` below.

Reply:
Refactored to introduce `skip_if_no_ssl_dir`, which made the repeated calls to `getenv` in these tests obsolete.